Routers/usersRouter.js

```diff
@@ -0,0 +1,77 @@
+const userDb = require('../data/helpers');
+
```

Comment:
This looks great Christopher 👍 code is clean, and comments are acting as documentation.